test/automation/projects_automation_test.go

```diff
@@ -0,0 +1,87 @@
+package automation
+
+import (
+	"fmt"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/ActiveState/cli/internal/testhelpers/tagsuite"
+	"github.com/stretchr/testify/suite"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+type ProjectsAutomationTestSuite struct {
+	tagsuite.Suite
+}
+
+func (suite *ProjectsAutomationTestSuite) TestProjects_NoActProjects() {
+	suite.OnlyRunForTags(tagsuite.Automation)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("projects")
+	cp.ExpectLongString("You have not activated any projects yet")
+	cp.Wait(5 * time.Second)
```

Comment:
Rather than wait here you could expect an exit code or just do nothing as the last expect might cover everything you're looking to test